misc/ls-extensions/modules/performance-analyzer-services/build.gradle

```diff
@@ -29,22 +31,22 @@ dependencies {
     compileOnly project(':ballerina-lang')
     compileOnly project(':ballerina-tools-api')
     compileOnly project(':language-server:language-server-commons')
-    implementation ("io.swagger.parser.v3:swagger-parser-v2-converter") {
+    implementation(libs.swagger.parser.v2) {
```

Comment:
it seems we we are using V2 now against V3 which we had earlier.

Reply:
Thanks, I will fix that 👍🏻

Reply:
@gimantha I'm a bit confused what you mean. This PR uses the are same version as on master.

Reply:
> @gimantha I'm a bit confused what you mean. This PR uses the are same version as on master.

my bad.. found the lib file where you have defined the versions